modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -2245,6 +2215,14 @@ private AccountEntry _addAccountEntry() throws Exception {
 			ServiceContextTestUtil.getServiceContext());
 	}
 
+	private void _addAccountEntryOrganizationRel(
+			AccountEntry accountEntry, Organization organization)
+		throws Exception {
```

Comment:
What do you think about changing the method signature to 

```
private void _addAccountEntryOrganizationRel(
			long accountEntryId, long organizationId)
		throws Exception {
```
?

Reply:
Hey @seltong

If we change to  `long accountEntryId, long organizationId` we will need two lines to call the method. I prefer pass all entity an use just one line in call. See same [here](https://github.com/liferay/liferay-portal/blob/53891c022ac74ff0a55e3a6da7dc738340966266/modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java#L2288).